source/codegen/metadata/nirfmxnr_restricted/config.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+config = {
+    "code_readiness": "RestrictedRelease",
```

Comment:
If/When you get back to this a heads up that I changed the restricted proto / service exposure a little bit with #842 

I changed the config.py expectations to separate the restricted and code_readiness concept so it now expects it in the form
```py
    "code_readiness": "Release",
    "is_restricted": True,
```

Reply:
Thanks for letting me know, but I actually forgot to abandon this PR. Since we added the functionality we needed in the public server, there is no longer a need for a restricted service for NR.